Jenkinsfile

```diff
@@ -59,6 +59,17 @@ pipeline {
             deleteDir()
             gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true)
             stash allowEmpty: true, name: 'source', useDefaultExcludes: false
+            // Look for changes related to the benchmark, if so then set the env variable.
+            script {
+              dir("${BASE_DIR}"){
+                def regexps =[
+                  '^packages/rum/test/benchmarks/',
```

Comment:
```suggestion
                  '^packages/**/test/benchmarks/',
```

Reply:
Yeah sounds good to me. I dont see any huge concern on this.